core/bc_chtransfer.go

```diff
@@ -144,6 +194,93 @@ func (bc *BaseContract) createCCTransferFrom(
 	return bc.GetStub().GetTxID(), nil
 }
 
+func (bc *BaseContract) createMultiCCTransferFrom(
+	idTransfer string,
+	to string,
+	idUser *types.Address,
+	items []TransferItem,
+) (string, error) {
+	if strings.EqualFold(bc.ContractConfig().GetSymbol(), to) {
+		return "", cctransfer.ErrInvalidChannel
+	}
+
+	if len(items) == 0 || len(items) > bc.getMaxChannelTransferItems() {
+		return "", fmt.Errorf("%w found %d but expected from 1 to %d",
+			cctransfer.ErrInvalidTransferItemsCount, len(items), bc.getMaxChannelTransferItems(),
+		)
+	}
+
+	t := tokenSymbol(items[0].Token)
+	if !strings.EqualFold(bc.ContractConfig().GetSymbol(), t) && !strings.EqualFold(to, t) {
+		return "", cctransfer.ErrInvalidToken
+	}
```

Comment:
maybe check all the tokens?

Reply:
I check all tokens symbol is eq with symbol items[0]